contract/src/test/java/io/neow3j/contract/PolicyContractTest.java

```diff
@@ -6,8 +6,7 @@
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.*;
```

Comment:
wildcard?! noooooooooooooooooooooooo

🤣🤣🤣🤣🤣🤣🤣🤣🤣

Reply:
I set up my laptop from scratch and forgot to disable wildcards in the my IDE when I installed it again...